src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/PhysicalAddress.cs

```diff
@@ -103,22 +103,41 @@ public byte[] GetAddressBytes()
             return (byte[])_address.Clone();
         }
 
-        public static PhysicalAddress Parse(string address)
+        public static PhysicalAddress Parse(string address) => Parse(address.AsSpan());
+
+        public static PhysicalAddress Parse(ReadOnlySpan<char> address)
+        {
+            if (!TryParseInternal(address, out PhysicalAddress value))
+                ThrowBadAddressException(address);
```

Comment:
nit: curly brackets

Reply:
AFAIK, they are not required. Are they?

Reply:
Uh..I thought it was a default style, you did it also below.

Reply:
It's not required, so long as it isn't nested in another control block.